src/source/feeds.rst

```diff
@@ -13,16 +13,63 @@ We treat the provided XML file as the desired list of ads the user wants to have
+:ref:`feed_priceType`                  sales model for product              enum                 yes       
+:ref:`feed_price`                      product price in cents if applicable positive integer     yes/no       
+:ref:`feed_originalPrice`              original price before discount       positive integer     no        
+:ref:`feed_media`                      product images                       :ref:`feed_media`    no
+:ref:`feed_attr`                       collection of ad attributes          :ref:`feed_attr`     no        
+:ref:`feed_budget`                     budget details                       :ref:`feed_budget`   no        
+:ref:`feed_ship`                       shipping options                     :ref:`feed_ship`     no
+:ref:`feed_phoneNumber`                phone number                         max. 32 chars        no        
+:ref:`feed_emailAdvertiser`            allow emails to the seller           true,false           no
+:ref:`feed_regionId`                   only applicable for Kijiji Canada    numeric              no        
+:ref:`feed_engines`                    targeted search engines for this ad  emunerated list      no
+:ref:`feed_microTip`                   tiny product hightlight              max. 18 chars        no
+:ref:`feed_mpn`                        Manufacturer Part Number (MPN)       len. 2-70 chars      no   
+:ref:`feed_googleProductCategory`      google category for your product     string               no
+:ref:`feed_productType`                customer product type                max. 750 chars       no    
+:ref:`feed_brand`                      product brand name                   max. 70 chars        no
+:ref:`feed_gtin`                       Global Trade Identification Number   max. 50 chars        no  
+:ref:`feed_itemGroupId`                groups product variants in your      len. 1-50 chars      no
+:ref:`feed_condition`                  condition of product                 enum                 no
+:ref:`feed_material`                   main product fabric or material      max. 200 chars       no
+:ref:`feed_energyEfficiencyClass`      energy efficiency class              enum                 no
+:ref:`feed_minEnergyEfficiencyClass`   minimal energy efficiency class      enum                 no
+:ref:`feed_maxEnergyEfficiencyClass`   maximal energy efficiency class      enum                 no
+:ref:`feed_color`                      product color                        len. 1-100 chars     no
+:ref:`feed_gender`                     gender product is designed for       enum                 no
+:ref:`feed_ageGroup`                   age group product is intended for    enum                 no
+:ref:`feed_size`                       size information                     enum                 no
+:ref:`feed_unitPricingBaseMeasure`     denominator for product unit price   string               no
+:ref:`feed_unitPricingMeasure`         measure and dimension of product     string               no
```

Comment:
the description is about dimension & measuers, but the field is called _unitPricing*  making it hint that this is about the item price. Is this correct?

Reply:
@twessling-icas that's correct. the two unit pricing fields (along with the price field) are used to calculate price/unit for those products where it is legally required to display that info (e.g. windshield wiper fluid), rather than have the seller calculate it themselves. The BaseMeasure field is the base unit to display (e.g. 100 ml) while the PricingMeasure field is the measure of the product (e.g. 973 ml)

Reply:
aah, I see. That's actually kinda nice :)